.travis.yml

```diff
@@ -10,11 +10,14 @@ env:
 rvm:
   - 2.6.2
   - 2.3.1
+addons:
+  postgresql: 9.4
```

Comment:
Can you say why you added this @jtapia ?

Reply:
this a fix for this issue https://travis-ci.org/collectiveidea/awesome_nested_set/jobs/540578878,  Rails 6 drops support for PostgreSQL version less than 9.3 https://github.com/rails/rails/pull/34520